mesonbuild/modules/python.py

```diff
@@ -759,7 +759,7 @@ def find_installation(self, state: 'ModuleState', args: T.Tuple[T.Optional[str]]
         if python.found() and not missing_modules:
             msg.extend([mlog.green('YES'), '({})'.format(' '.join(python.command))])
         else:
-            msg.append(mlog.red('NO'))
+            msg.extend([mlog.red('NO'), 'missing: {}'.format(', '.join(missing_modules))])
```

Comment:
@eli-schwartz reduced this down to the simplest change.

Reply:
I think we only want to extend it with the "missing" text if `missing_modules`, but this branch is also taken when `python.found()` is true.

So perhaps:
```python
msg.append(mlog.red('NO'))
if not missing_modules:
    msg.append('missing: {}'.format(', '.join(missing_modules)))
```